index.html

```diff
@@ -0,0 +1,179 @@
+                    </div>
+                </div>
+                <div class="program-card">
+                    <div class="card-icon">
+                        <img src="images/icons/icon2.svg" alt="icon 2">
+                        <h3>Professionalism</h3>
+                    </div>
+                    <div class="card-description">
+                        <p>We are dedicated to customers satisfaction</p>
+                    </div>
+                </div>
+                <div class="program-card">
+                    <div class="card-icon">
+                        <img src="images/icons/icon3.png" alt="icon 3">
+                        <h3>Perfect precision </h3>
+                    </div>
+                    <div class="card-description">
+                        <p>We are always untop of the game, when it comes to service delivery</p>
+                    </div>
+                </div>
+                <div class="program-card">
+                    <div class="card-icon">
+                        <img src="images/icons/icon4.png" alt="icon 4">
+                        <h3>Modern design</h3>
+                    </div>
+                    <div class="card-description">
+                        <p>We are sensitive to trends and we keep up with current innovations</p>
+                    </div>
+                </div>
+                <div class="program-card">
+                    <div class="card-icon">
+                        <img src="images/icons/icon5.png" alt="icon 5">
+                        <h3>Featured in magazines</h3>
+                    </div>
+                    <div class="card-description">
+                        <p>We have created a lot of websites for several businesses</p>
+                    </div>
+                </div>
+            </div>
+            <div class="program-btn">
+                <button type="button">Join us at 3ple Tech for your software development</button>
+            </div>
+        </section>
+
+    </main>
+
+    <section id="testimonials">
+        <div class="testimonial-title">
+          <h2 class="testimonial-header">Testimonials</h2>
+          <hr>
+        </div>  
+        <div class="testimonial-container">
+        </div>
+    </section> 
+
+    <div id="about-partners">
+      <div class="about-wrapper-partners">
+        <h2>Partners</h2>
+        <hr>
+      </div>
+      <div class="about-partners-icons">
+        <div class="about-p-icons">
+          <img src="./images/mozilla.png" class="partners-img" alt="mozilla">
+          <img src="./images/googlekorea.png" class="partners-img" alt="google">
+          <img src="./images/naver.png" class="partners-img" alt="naver">
+        </div>
+        <div class="about-p-icons">
+          <img src="./images/kakao-2.png" class="partners-img" alt="kakao">
+          <img src="./images/airbnb-2.png" class="partners-img" alt="airbnb">
+        </div>
+      </div>
+    </div>
+    
+    <footer id="footer">
```

Comment:
Kindly note that your footer does not show in the home page for mobile version. Kindly fix this as its part of the page.

Reply:
- [ ] This issue is well resolved :heavy_check_mark: